bundler/lib/bundler/cli/gem.rb

```diff
@@ -448,19 +447,6 @@ def standard_version
       "1.3"
     end
 
-    # TODO: remove at next minor release
-    def travis_removal_info
-      if options[:ci] == "travis"
-        Bundler.ui.error "Support for Travis CI was removed from gem skeleton generator."
-        exit 1
-      end
-
-      if Bundler.settings["gem.ci"] == "travis"
```

Comment:
This can cause troubles for people having this in config file (not passing travis as an argument on cli) breaking commands. :thinking: Are we sure this is ok to do now? Was this deprecation present already in bundler included with Ruby 3.2?

Reply:
>Was this deprecation present already in bundler included with Ruby 3.2?


According to [Ruby 3.2.0 release note](https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released/#:~:text=bigdecimal%203.1.3-,bundler%202.4.1,-cgi%200.3.6), Ruby 3.2.0 includes bundler 2.4.1 that include the deprecation. 👀

Reply:
@simi When I checked, the code was 2 years old, so it's been through at least 2 minor bundler releases.

Reply:
OK, all good. Thanks for check!